int8 pattern
int8 tonea
int8 toneb
int8 duration
bool reset